Mathlib/GroupTheory/Perm/Cycle/Basic.lean

```diff
@@ -220,7 +220,7 @@ section IsCycle
 
 variable {f g : Perm α} {x y : α}
 
-/-- A cycle is a non identity permutation where any two nonfixed points of the permutation are
+/-- A cycle is a nonidentity permutation where any two nonfixed points of the permutation are
```

Comment:
dito regarding hyphens

Reply:
Changed to non-identity because I didn't bother to do more research, and "non-identity" was more popular anyway.